packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.0.1]
+
+### Changed
+
+- Bump `@metamask/keyring-controller` to ^16.0.0 ([#4234](https://github.com/MetaMask/core/pull/4171))
```

Comment:
This is a peer dependency bump, making this a breaking change

Reply:
Likewise for the other packages with `peerDependency` changes (e.g. `@metamask/assets-controllers`)